### PR TITLE
UHM-6893: "Saved Forms" filtering not working when in French for some…

### PR DESCRIPTION
**Pull request by mogoodrich**
… forms

### CODE REVIEW

**Comment by mogoodrich** [comment]
Swapped quotes with double quotes...

**Review by cioan — APPROVED** [review]
Thanks @mogoodrich !

File: omod/src/main/webapp/pages/patient/encounterList.gsp [hunk 1]
```diff
@@ -89,6 +89,7 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient.patient ])
 
         jq("#encounter-type-filter").change(function() {
            let typeName = jq(this).val();
+            console.log(jq(this).val());
```
Comment (mseaton) [inline omod/src/main/webapp/pages/patient/encounterList.gsp:92]: I assume this log statement was for debugging?  Do you want this committed?

**Review by mseaton — APPROVED** [review]
See one comment.

File: omod/src/main/webapp/pages/patient/encounterList.gsp (hunk 1 above)
Comment (mogoodrich) [reply omod/src/main/webapp/pages/patient/encounterList.gsp:92]: Bad!  lazy on my part, thanks, removing!

**Pull request merged**